### PR TITLE
RavenDB 8222: No Fix, just improvements found while investigating the issue.

### PR DESCRIPTION
**Pull request by redknightlois**

### CODE REVIEW

File: src/Raven.Client/Raven.Client.csproj [hunk 1]
```diff
@@ -23,6 +23,10 @@
     <GenerateNeutralResourcesLanguageAttribute>false</GenerateNeutralResourcesLanguageAttribute>
     <Product>RavenDB Client</Product>
     <Title>RavenDB Client</Title>
+    <Configurations>Debug;Release;Validate</Configurations>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Validate|netstandard1.3|AnyCPU'">
```
Comment (ppekrol) [inline src/Raven.Client/Raven.Client.csproj]: This will probably not be applied for netstandard2.0

File: src/Sparrow/Sparrow.csproj [hunk 2]
```diff
@@ -23,6 +24,13 @@
   <PropertyGroup Condition=" '$(Configuration)' == 'Validate' ">
     <DefineConstants>$(DefineConstants);MEM_GUARD_STACK</DefineConstants>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Release|netstandard1.3|AnyCPU'">
+    <DefineConstants>TRACE;RELEASE;NETSTANDARD1_3</DefineConstants>
+  </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Validate|netstandard1.3|AnyCPU'">
+    <DefineConstants>TRACE;RELEASE;NETSTANDARD1_3;VALIDATE;</DefineConstants>
```
Comment (ppekrol) [inline src/Sparrow/Sparrow.csproj]: IIRC you can use:
```
<DefineConstants>$(DefineConstants);VALIDATE;</DefineConstants>
```

This also will be applied only to netstandard1.3

File: src/Raven.Client/Raven.Client.csproj (hunk 1 above)
Comment (redknightlois) [reply src/Raven.Client/Raven.Client.csproj]: Good catch. I  did this when we were already on netstandard2.0, wondering why it didnt applied it at the same time.

Comment (ppekrol) [reply src/Raven.Client/Raven.Client.csproj]: Probably removing '$(TargetFramework)' from condition will work for you.

Comment (ppekrol) [reply src/Raven.Client/Raven.Client.csproj]: Note that Sparrow and Client are compiling for netstandard1.3 and 2.0 now.

File: src/Sparrow/Sparrow.csproj (hunk 2 above)
Comment (redknightlois) [reply src/Sparrow/Sparrow.csproj]: Apparently it copied it from the Release which has:
``` 
<DefineConstants>TRACE;RELEASE;NETSTANDARD1_3</DefineConstants>
```

**Comment by redknightlois** [comment]
Changed that and amended the commit.

**Comment by redknightlois** [comment]
@ppekrol Took a few ammends to correct all small details. Check it now.

**Comment by ppekrol** [comment]
LGTM

**Pull request merged**